checks/readme.php

```diff
@@ -3,6 +3,15 @@
 use WordPressdotorg\Plugin_Check\{Error, Guideline_Violation, Message, Notice, Warning};
 
 class Readme extends Check_Base {
+	public function check_readmetxt_exists() {
+		if ( ! $this->readme ) {
+			return new Error(
+				'readme_missing',
+				'No readme.txt or readme.md was not found. readme.txt/readme.md is a required file.'
+			);
+		}
```

Comment:
Invert the conditional and bail to reduce cognitive load for long-term maintenance.
Also translations.
```suggestion
		if ( $this->readme ) {
			return;
		}
		return new Error(
			'readme_missing',
			__( 'No readme.txt or readme.md was not found. readme.txt/readme.md is a required file.', 'wporg-plugins' )
		);
```